builders/xcm/fees.md

```diff
@@ -261,18 +277,118 @@ ExecutionTime = ({{ networks.moonbeam.xcm.instructions.weight_units.numbers_only
 
 Which means that four XCM instructions cost `{{ networks.moonbeam.xcm.message.transfer.exec_time }}` seconds of block execution time.
 
-To calculate the total cost in xcDOT, you'll also need the number of decimals the asset in question uses, which for xcDOT is 10 decimals. You can determine the number of decimals for any asset by [querying the asset metadata](/builders/xcm/xc20/xc20/#x-chain-assets-metadata){target=_blank}.
+To calculate the total cost in xcDOT, you'll also need the number of decimals the asset in question uses, which for xcDOT is ten decimals. You can determine the number of decimals for any asset by [querying the asset metadata](/builders/xcm/xc20/xc20/#x-chain-assets-metadata){target=_blank}.
 
 The block execution formula can then be used to determine how much Alice's transfer of DOT to Alith's account on Moonbeam costs. The formula for finding the total cost is as follows:
 
 ```
-Cost = (UnitsPerSecond / DecimalConversion) * ExecutionTime
+XCM-Cost = (UnitsPerSecond / DecimalConversion) * ExecutionTime
 ```
 
 Then the calculation for the transfer is:
 
 ```
-Cost = ({{ networks.moonbeam.xcm.units_per_second.xcdot.numbers_only }} / 10^10) * {{ networks.moonbeam.xcm.message.transfer.exec_time }}
+XCM-Cost = ({{ networks.moonbeam.xcm.units_per_second.xcdot.transfer_numbers_only }} / 10^10) * {{ networks.moonbeam.xcm.message.transfer.exec_time }}
+```
+
+The total cost to transfer Alice's DOT to Alith's account for xcDOT is `{{ networks.moonbeam.xcm.message.transfer.xcdot_cost }} xcDOT`.
+
+## XCM-Transactor Fees
+
+The [XCM-transactor pallet](/builders/xcm/xcm-transactor/) builds an XCM message to remotely transact in other chains of the ecosystem. 
+
+There are two different ways developers can remotely transact through the pallet:
+
+1. [`transactThroughDerivative`](/builders/xcm/xcm-transactor/#xcmtransactor-transact-through-derivative){target=_blank}
+2. [`transactThroughSigned`](/builders/xcm/xcm-transactor/#xcmtransactor-transact-through-signed){target=_blank}, in which the dispatcher account in the destination chain is a multilocation-derived account and must have enough funds to cover XCM execution fees plus whatever call is being remotely executed
+
+Generally speaking, the XCM instructions normally involved for remote execution are:
+
+ - Instructions to handle tokens in the origin chain. This can be either moving tokens to a sovereign account or burning the corresponding [XC-20](/builders/xcm/xc20/overview/){target=_blank} so that it can be used in the target chain. These instructions get executed in the origin chain
```

Comment:
is this meant to be here? seems like it was just supposed to be a list of the actual instructions

Reply:
Yes, what I meant here is that the first instruction is always something to handle tokens in the origin chain. They differ depending if it is a Reserve asset or not. Any suggestions?

Reply:
Maybe start with just that... "The first instruction handles tokens in the origin chain. This can be..."

Reply:
Thanks, I like that